src/main/resources/db/changelog/changes/v1.1.0/update-drv-purchase-order-line-details-definition.xml

```diff
@@ -333,6 +333,10 @@
               "valueGetter": "src_purchase_order_line.jsonb -> 'cost' ->> 'currency'",
               "filterValueGetter": "lower(src_purchase_order_line.jsonb -> 'cost' ->> 'currency')",
               "valueFunction": "lower(:value)",
+              "source": {
+                "entityTypeId": "90403847-8c47-4f58-b117-9a807b052808",
+                "columnName": "pol_currency"
+              },
```

Comment:
This is kind of meaningless, but I had to add it in so that the query builder sends the request to GET /columns/pol_currency/values. For it to call that API, it's expecting either a "values", "source" or a "valueSourceApi" field here. POL currency doesn't actually use any of these, but we still need one present to get the call to happen.

Reply:
Yeah, I've run into this, too :/ It's a little gross, but also makes a small amount of sense (it tells the UI how to ask for the values). I'd definitely prefer that there only be 1 way for it to get the values, though, rather than have it depend on `source`